website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -96,6 +96,16 @@ timestamp appended.
 
 ## Nomad 1.0.10 and 1.1.4
 
+#### Strict mTLS certificate check
+
+Raft requests were not being verified for their certificate common alternative
+name. A check was introduced to make sure only servers in the same region are
```

Comment:
> their certificate common alternative name.

Did you mean `Common Name` or `Subject Alternative Name`? Or maybe I'm forgetting a term.